modules/system.js

```diff
@@ -208,6 +224,13 @@ System.prototype.update = function (cb) {
 		getHeight: function (seriesCb) {
 			__private.height = modules.blocks.lastBlock.get().height;
 			return setImmediate(seriesCb);
+		},
+		setHeaders: function (seriesCb) {
+			constants.setConst('headers', _.assign(constants.headers || {}, {
```

Comment:
ok I agree. The better idea could be to pass getter for the headers from system module when it is necessary. Do you agree?